Clojure/syntax_test_clojure.clj

```diff
@@ -1,107 +1,666 @@
-;;     ^^^^^^^^ entity.name.function.clojure source.symbol.clojure
-;;              ^^^^^^^^^^^^^^^^^^^^^ meta.function.lambda.clojure
-;;               ^^ storage.type.function.type.clojure
-;;                  ^^^^^^^^^^^^^^^^ meta.function.body.clojure
-;;                  ^^ meta.function.parameters.vector.clojure punctuation.definition.vector
-;;                     ^^^^^^^^^^^^^ meta.function.body.code.clojure string.quoted.double.clojure
-;;                                  ^ - meta.function.body.code.clojure string.quoted.double.clojure
-;;                     ^ punctuation.definition.string.begin.clojure
-;;                                 ^ punctuation.definition.string.end.clojure
-
-  (def hello1 (fn [] #"Hello world"))
-;;                   ^^^^^^^^^^^^^^ string.regexp.clojure
-;;                   ^^ punctuation.definition.string.begin.clojure
-;;                                ^ punctuation.definition.string.end.clojure
-;;                                 ^ - string.regexp.clojure
-;;                     ^^^^^^^^^^^ meta.literal.regexp
-
-  (def hello2 (fn [] #"Hello world\\"))
-;;                   ^^^^^^^^^^^^^^^^ string.regexp.clojure
-;;                   ^^ punctuation.definition.string.begin.clojure
-;;                                  ^ punctuation.definition.string.end.clojure
-;;                                   ^ - string.regexp.clojure
-;;                     ^^^^^^^^^^^ meta.literal.regexp
-;;                                ^^ constant.character.escape.regexp
-
-  (def hello (fn [] #"{2}Hello*?* world\"{2} foobar*"))
-;;                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ string.regexp.clojure
-;;                  ^^ punctuation.definition.string.begin.clojure
-;;                                     ^^ constant.character.escape.regexp - punctuation.definition.string.end.clojure
-;;                                                  ^ punctuation.definition.string.end.clojure
-;;                                                   ^ - string.regexp.clojure
-;;                       ^^^^^ meta.literal.regexp
-;;                                          ^^^^^^^ meta.literal.regexp
-;;                    ^^^ invalid.illegal.unexpected-quantifier.regexp - meta.literal.regexp
-;;                            ^^ keyword.operator.quantifier.regexp
-;;                              ^ invalid.illegal.unexpected-quantifier.regexp
-;;                                       ^^ keyword.operator.quantifier.regexp
-;;                                                 ^ keyword.operator.quantifier.regexp
-
-  (def hello (fn [] #"(?=unclosed_paren"))
-;;                  ^^^^^^^^^^^^^^^^^^^^ string.regexp.clojure
-;;                  ^^ punctuation.definition.string.begin.clojure
-;;                                     ^ punctuation.definition.string.end.clojure
-;;                                      ^ - string.regexp.clojure
+; SYNTAX TEST "Packages/Clojure/Clojure.sublime-syntax"
```

Comment:
The point of these syntax test files is that you *test* the syntax. For example,
```
"blah \" blah"
; <- punctuation.section
; ^ string.quoted.double
```
Put your build system on "Automatic" and hit "Build" to run these two tests.